config/routes.rb

```diff
@@ -37,6 +37,4 @@
   authenticated :user do
     mount DelayedJobWeb, at: "/delayed_job"
   end
-  # fall back if not authenticated
-  get '/delayed_job', to: redirect('/management/users/sign_in')
```

Comment:
I am pretty sure this should be `get '/delayed_job', to: redirect('/management/users/auth/cas')`

Reply:
Fixed